canonical/landscape-client
# Add livepatch to manager instead of monitor, refactored

Manual Testing Instructions:

This will require a client with this code and a server with the linked below code running so that you can see the new messages.

1) Check out the server branch https://github.com/canonical/landscape-server/pull/82 and run server
2) Start server
3) Change the code in client under landscape/client/manager, the files for livepatch, ubuntu pro, and pro reboot. Change the code so that it runs every 60 seconds so you can see it
4) (Optional) Comment out some of the other noisy plugins in landscape/client/monitor/config.py and landscape/client/manager/config.py to make logs cleaner
5) Register client to the server and accept the computer
6) Monitor the client logging to see messages are getting sent and cached from ubuntu pro, reboot required, and livepatch
7) Send api requests to` http GET "localhost:9091/v2/computer/ID" Authorization:"Bearer $JWT"` so that you can see the new messages

Note a separate endpoint will be created to clean up the output. This is just testing that the data is going through
